pkgs/applications/misc/far2l/default.nix

```diff
@@ -67,6 +67,8 @@ stdenv.mkDerivation rec {
       mkdir -p $out/share/icons/hicolor/$size/apps
       convert -size $size ../far2l/DE/icons/hicolor/$size/apps/far2l.svg $out/share/icons/hicolor/$size/apps/far2l.png
     done
+  '' + stdenv.lib.optionalString stdenv.isDarwin ''
+    wrapProgram $out/bin/far2l --argv0 $out/bin/far2l
```

Comment:
Nice workaround, but ugly bug to begin with. Do you think upstream could fix this anytime soon?

Reply:
Maybe.
It is not specific to nixpkgs, homebrew package has this problem too